website/config/routes.js

```diff
@@ -15,7 +15,8 @@ module.exports.routes = {
   //  ╚╩╝╚═╝╚═╝╩  ╩ ╩╚═╝╚═╝╚═╝
   'GET /':                   { action: 'view-homepage-or-redirect', locals: { isHomepage: true } },
   'GET /company/contact':    { action:   'view-contact' },
-  'GET /get-started':        { action:   'view-pricing' },
+  'GET /get-started':        { action:   'view-get-started' },
+  'GET /pricing':            { action: 'view-pricing'},
 
   'GET /install':            { action:   'view-get-started' },
```

Comment:
Could you move `GET /install` down to "Convenience" in the misc redirects section?

i.e. it becomes:
`'/install':                 '/get-started',`

Reply:
This one is confusing me, sorry. I added `'/install': '/get-started',` to "Convenience", and removed `'GET /install':            { action:   'view-get-started' },` from "Webpages". Not sure if the latter is correct?